src/gmail.js

```diff
@@ -1247,6 +1247,13 @@ var Gmail = function(localJQuery) {
         return res;
     };
 
+    api.tools.check_fd_attachment_url = function(url) {
+        var userAccountUrlPart = api.tracker.globals[7];
+        if (url.indexOf(userAccountUrlPart) < 0) {
```

Comment:
Should we handle the possibility of url being null or empty here?

It may not be needed, but in case of code running inside hijacked XHR requests, I'd rather be safe than sorry.